lua/orgmode/capture/template/init.lua

```diff
@@ -14,7 +16,7 @@ local expansions = {
     return vim.fn.expand('%:p')
   end,
   ['%%n'] = function()
-    return os.getenv('USER')
+    return is_windows and os.getenv('USERNAME') or os.getenv('USER')
```

Comment:
Lets just do everything from inside this function like this and remove the `is_windows` variable:
```lua
if vim.fn.has('win32') == 1 then
  return os.getenv('USERNAME')
end
return os.getenv('USER')
```